plugin/pkg/scheduler/apis/schedulerconfig/v1alpha1/register.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2015 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// GroupName is the group name use in this package
+const GroupName = "kubescheduler.k8s.io"
```

Comment:
This should probably either be scheduler.config.k8s.io or kubescheduler.config.k8s.io. Given that for kube-proxy we have kubeproxy.config.k8s.io, I think kubescheduler.config.k8s.io makes sense. cc @mtaufen @mikedanese @lavalamp @liggitt @jbeda

Reply:
> I think kubescheduler.config.k8s.io makes sense

+1